common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +256,61 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    return (
+      isValidETHAddress(this.state.to) &&
+      this.state.value &&
+      !isNaN(Number(this.state.value)) &&
+      isFinite(Number(this.state.value))
+    );
+  }
+
+  // FIXME MOVE ME
```

Comment:
To move something somewhere, one has to think of a place to move it to and it is hard :D